### PR TITLE
[Merged by Bors] - chore: Rename `LocallyFiniteOrder` instances

### PR DESCRIPTION
**Pull request by YaelDillies**
The generated names were too long

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/PNat/Interval.lean [hunk 1]
```diff
@@ -18,8 +18,7 @@ intervals as finsets and fintypes.
 
 open Finset Function PNat
 
-instance : LocallyFiniteOrder ℕ+ :=
-  Subtype.instLocallyFiniteOrder _
+instance instLocallyFiniteOrder : LocallyFiniteOrder ℕ+ := Subtype.instLocallyFiniteOrder _
```
Comment (jcommelin) [inline Mathlib/Data/PNat/Interval.lean:21]:
```suggestion
instance PNat.instLocallyFiniteOrder : LocallyFiniteOrder ℕ+ := Subtype.instLocallyFiniteOrder _
```

File: Mathlib/Data/Nat/Interval.lean [hunk 2]
```diff
@@ -13,7 +13,7 @@ import Mathlib.Data.Finset.LocallyFinite.Basic
 This file proves that `ℕ` is a `LocallyFiniteOrder` and calculates the cardinality of its
 intervals as finsets and fintypes.
 
-## TODO
+## TOace
```
Comment (jcommelin) [inline Mathlib/Data/Nat/Interval.lean:16]:
```suggestion
## TODO
```

**Review by jcommelin — APPROVED** [review]
bors d+

**Comment by YaelDillies** [comment]
bors merge

**Pull request closed without merging**